program_indicator/security/ir_rule.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+  <data>
+
+    <record id="rule_indicator_value_director" model="ir.rule">
+        <field name="name">Program Indicator Value Director</field>
+        <field model="ir.model" name="model_id" ref="model_program_result_indicator_value"/>
+        <field name="domain_force">[('state', '=', 'draft')]</field>
+        <field name="groups" eval="[(4, ref('program.group_program_director'))]"/>
+        <field name="perm_read" eval="0"/>
+        <field name="perm_create" eval="0"/>
+        <field name="perm_unlink" eval="0"/>
+    </record>
+
+    <record id="rule_indicator_value_dpe" model="ir.rule">
+        <field name="name">Program Indicator Value DPE</field>
```

Comment:
@yvaucher, I agree 100% with you. This module was developed with security rules specific to the needs of one client. It does not fit in OCA in its current state. DPE is an expression used by one of Savoir-faire Linux's client. One day, I will propose another module to remove those security rules from OCA modules.